examples/1-interfaces/build.gradle

```diff
@@ -5,7 +5,8 @@ plugins {
 task pyang(type: JncPyangTask) {
     pluginDir = JNCDir
     outputDir = new File("${buildDir}/generated/src/gen")
-    inputFiles = [new File("ietf-interfaces.yang")]
+    yangPath = projectDir
+    inputFiles = [new File("$projectDir/ietf-interfaces.yang")]
```

Comment:
Forgot to comment before - shouldn't the `inputFiles` be just an array be of simple strings?
And then "convert" strings to `new File(...)` where needed - at the point of `inputFiles` usage.
This will reduce need of adding it for all array items here...